images/base/files/usr/local/bin/entrypoint

```diff
@@ -201,6 +201,22 @@ mount_kubelet_cgroup_root() {
   mount --bind "${subsystem}/${cgroup_root}" "${subsystem}/${cgroup_root}"
 }
 
+# helper used by fix_cgroup
+mount_kubelet_cgroup_root() {
```

Comment:
This is just pulled out from the end of fix_cgroup for re-use in v1 both with/without cgroupns.

We're responsible for setting up the cgroup when we use --cgroup-root.

We did that previously for the "v1 no cgroupns and nested in another cluster" reasons, but we'll need to keep `--cgroup-root=kubelet` until we can phase it out at some point later following #3241.